CenterForOpenScience/osf.io
# [WIP] [ENG-1679] Hotfix for Draft Registration Permissions

## Purpose
Addresses multiple bugs with draft registration permissions

## Changes
* Changes permission on NodeDraftRegistrationList to match DraftRegistrationList
* Changes permission for draft_registration related_counts (ADMIN to WRITE)

## QA Notes
These changes should be tested to verify that they properly fix the bugs in the JIRA ticket but also verify that other permission requirements around draft_registrations are in line with expectations.

## Documentation
Internal documentation tracking permissions should be updated appropriately

## Side Effects
N/A

## Ticket
[JIRA Ticket](https://openscience.atlassian.net/browse/ENG-1679)
